<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovesWidget</class>
 <widget class="QWidget" name="MovesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moves</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_GoBack1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Previous move</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/previous_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_GoBack2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Back 1 full move (2 plies)</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/previous_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_GoStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Skip to the beginning</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/first_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_GoNext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Next move</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/next_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btn_GoCurrent">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Current position</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/last_16x16"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_Moves" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
